### PR TITLE
Bitcoin 0.13.1

### PR DESCRIPTION
**Pull request by luke-jr**
@blueness

### CODE REVIEW

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	)
+"
+DEPEND="${RDEPEND}
+	qt5? ( dev-qt/linguist-tools:5 )
+	ljr? (
+		gnome-base/librsvg
+		media-gfx/imagemagick[png]
+	)
+"
+REQUIRED_USE="^^ ( qt4 qt5 )
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: what @blueness said, of course I meant Qt stuff here ;)

Comment (luke-jr) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: I dislike taking choices away from users. Do what you want in the main tree, but I don't agree with it.

Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: For reference: https://wiki.gentoo.org/wiki/Project:Qt/Policies#Handling_different_versions_of_Qt

**Comment by blueness** [comment]
Okay this is annoying.  So is this going to just sit here without any resolution to the qt4/5 issue?

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	dev-libs/protobuf
+	qrcode? (
+		media-gfx/qrencode
+	)
+	qt4? ( dev-qt/qtcore:4[ssl] dev-qt/qtgui:4 )
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: !qt5

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	qt5? ( dev-qt/qtgui:5 dev-qt/qtnetwork:5 dev-qt/qtwidgets:5 )
+	dbus? (
+		qt4? ( dev-qt/qtdbus:4 )
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: !qt5

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,130 @@
+# Copyright 2010-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+BITCOINCORE_COMMITHASH="03422e564b552c1d3c16ae854f8471f7cb39e25d"
+BITCOINCORE_LJR_DATE="20161027"
+BITCOINCORE_IUSE="dbus kde +libevent ljr +qrcode qt4 qt5 +http test +tor upnp +wallet zeromq"
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: drop USE=qt4

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+	einfo "Languages -- Enabled:$yeslang -- Disabled:$nolang"
+
+	bitcoincore_autoreconf
+}
+
+src_configure() {
+	bitcoincore_conf \
+		$(use_with dbus qtdbus)  \
+		$(use_with qrcode qrencode)  \
+		--with-gui=$(usex qt5 qt5 qt4)
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: no change necessary there, as only qt5 flag is used anyway

File: net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+	dodoc doc/assets-attribution.md doc/bips.md doc/tor.md
+	doman contrib/debian/manpages/bitcoin-qt.1
+
+	use zeromq && dodoc doc/zmq.md
+
+	if use kde; then
+		insinto /usr/share/kde4/services
+		doins contrib/debian/bitcoin-qt.protocol
+	fi
```
Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: Please add the file also to the relevant kf5 directory - that would be ``/usr/share/kservices5``

Comment (luke-jr) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: Just `insinto /usr/share/kservices5` and the same `doins`?

Comment (a17r) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.13.1.ebuild]: You could just use ``dosym`` to create a link in the other dir.

**Comment by luke-jr** [comment]
@a17r Look good now?

**Comment by a17r** [comment]
I didn't test, but the changes lgtm.

**Comment by blueness** [comment]
okay pushed

**Pull request closed without merging**